R/shiny.R

```diff
@@ -2166,6 +2166,13 @@ flushPendingSessions <- function() {
 }
 
 .globals$onStopCallbacks <- Callbacks$new()
+# Close ShinySession on.exit event (SIGINT signal)
+.globals$onStopCallbacks$register(function() {
+  lapply(appsByToken$keys(), function(session) {
+    session <- appsByToken$get(session)
+    if (isFALSE(session$closed)) session$wsClosed()
```

Comment:
isFALSE() is a function introduced by R3.5.0 so it's better to use `identical(session$closed, FALSE)`

Reply:
that's true, thanks for catching this.